<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRC калькулятор</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>611</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Введите данные через пробелы или запятые:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="inputTextEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Вот какие числа распознались:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="outputTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>211</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="crcLbl">
       <property name="text">
        <string>Значение CRC:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="crclineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="crcLbl_2">
       <property name="text">
        <string>Сумма:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="crcLbl_3">
       <property name="text">
        <string>Произведение:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sumLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="MultLineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="fileLoadAction"/>
    <addaction name="fileSaveAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="settingsMenu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="settingsAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_11"/>
    <addaction name="AboutAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="settingsMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="fileLoadAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="fileSaveAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>закрыть</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Формат входных данных</string>
   </property>
  </action>
  <action name="action_CRC">
   <property name="text">
    <string>Тип CRC</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Матерый</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Вежливый</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Грубиянский</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="AboutAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionHex">
   <property name="text">
    <string>Hex</string>
   </property>
  </action>
  <action name="actionDec">
   <property name="text">
    <string>Dec</string>
   </property>
  </action>
  <action name="actionBin">
   <property name="text">
    <string>Bin</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
